frontend/src/App.css

```diff
@@ -36,3 +36,5 @@
     transform: rotate(360deg);
   }
 }
+
```

Comment:
Can you remove the two new lines here? That way we can lessen the number of files you changed.